src/pages/Home.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-fragment>
+  <div>
```

Comment:
This was throwing a warning. Did not find fast a way to remove it, so decided just to change to a usual div 🤷‍♂